config/schedule.rb

```diff
@@ -1,3 +1,5 @@
+env 'PATH', '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
```

Comment:
So my question is: Since this means we'll have to add a line like this to every app we have running on the server, why did you choose not to simply configure cron itself? Because this fix means we'll have to just find a way to tell any future developers "by the way, if you want a cron job to run and your app is deployed on this server you have to set the path manually inside your app" which seems wrong to me.